policyengine_us/parameters/gov/states/md/tax/income/credits/eitc/refundable/montgomery/match.yaml

```diff
@@ -0,0 +1,18 @@
+description: Montgomery County matches this percentage of the refundable Maryland Earned Income Tax Credit. 
+values:
+  2015-01-01: 0.9
+  2016-01-01: 0.95
+  2020-01-01: 1
+  2023-07-01: 0.56 
+metadata:
+  unit: /1
+  period: year  
+  label: Maryland montgomery refundable state earned income tax credit match
+  reference: 
+    - title: 2023 Montgomery County Earned Income Credit EIC Refund
```

Comment:
could add comments to the individual sources adding e.g. that the 2023 change is not yet reflected in the legal code